spk/pyload/src/app/config

```diff
@@ -0,0 +1,12 @@
+{
+    ".url": {
+        "com.synocommunity.packages.pyload": {
+            "title": "pyLoad",
+            "desc": "pyLoad",
+            "icon": "images/pyload-{0}.png",
+            "type": "url",
+            "url": "3rdparty/pyload/pyload.cgi",
```

Comment:
That's the case for SABnzbd/SickBeard/CouchPotato too and the port is fixed anyway. If one changes the port number he should know the implications of that so that's OK.

This is how we do to avoid useless overhead in the code.